metadata.json

```diff
@@ -25,6 +25,10 @@
     { "operatingsystem": "Scientific", "operatingsystemrelease": [ "6", "7" ] },
     { "operatingsystem": "Fedora",     "operatingsystemrelease": [ "19" ] },
     { "operatingsystem": "Debian",     "operatingsystemrelease": [ "6", "7" ] },
-    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] }
+    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] },
+    { "operatingsystem": "Windows",    "operatingsystemrelease": [ 
+        "Vista", "7", "8",
+        "2008", "2008 R2", "2012", "2012 R2"
+      ] }
   ]
```

Comment:
How does this look? I broke up the desktop versions and the server versions since the list is so long... it made it much easier to read.